android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java

```diff
@@ -67,7 +67,7 @@ public synchronized void updateActions(long mask, Map<String, Integer> options)
         if (options != null && options.containsKey("skipBackward") && (options.get("skipBackward") == 10 || options.get("skipBackward") == 5 || options.get("skipBackward") == 30)) {
             skipBackward = createAction("skip_backward_" + options.get("skipBackward").toString(), "Skip Backward", mask, PlaybackStateCompat.ACTION_REWIND, skipBackward);
         } else {
-            skipBackward = createAction("skip_backward_", "Skip Backward", mask, PlaybackStateCompat.ACTION_REWIND, skipBackward);
+            skipBackward = createAction("skip_backward_10", "Skip Backward", mask, PlaybackStateCompat.ACTION_REWIND, skipBackward);
```

Comment:
That's actually supposed to be there. There [isn't a `skip_backward_` icon](https://github.com/tanguyantoine/react-native-music-control/tree/master/android/src/main/res/drawable).

I don't think I get what is the issue...

Reply:
Yeah, the problem is that there is no icon that corresponds to skip_backward_, there are icons for skip_backward_5, skip_backward_10, and skip_backward_30. The new code is just setting the default icon to skip_backward_10 (the default icon for skipforward is skip_forward_10).

Reply:
I can create a few gifs later today to show what this code is doing.

Reply:
Oh, I don't know what I was thinking, I thought you removed the `10` instead of adding it :P

Thank you for the PR :)

Reply:
No worries :).